framework/src/org/checkerframework/common/aliasing/qual/MaybeAliased.java

```diff
@@ -24,6 +24,6 @@
 @DefaultQualifierInHierarchy
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.TYPE_PARAMETER, ElementType.TYPE_USE })
-@ImplicitFor(trees = { Tree.Kind.NULL_LITERAL }, typeNames = { java.lang.Void.class })
+@ImplicitFor(literals = { LiteralKind.NULL }, typeNames = { java.lang.Void.class })
```

Comment:
I'm not sure when the j.i.Void type is used.  If it is for implicit lower bounds, that would effect how they would be defaulted.